addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Log exception details in Out of Band XSS scan rule.
+- Removed misleading "Attack" from the Proxy Disclosure scan rule alerts (Issue 8556).
```

Comment:
This should not have been done in this PR.

Reply:
That's fair, I'll roll that back and do a separate one.

Reply:
Should be part of #5718 IMO.

Reply:
Yup that makes sense. Will migrate it.

Reply:
This one still needs to be reverted.